packages/sui-deploy/README.md

```diff
@@ -35,7 +51,7 @@ $ sui-deploy spa 'test-project' --now -p
 
 Deploys to test-project.now.sh
 
-#### `-a, --auth <user:password>`
+#### `-a, --auth <user:password>` (only for spa)
```

Comment:
I dont know how or if we have to say something here, but you can do the same with the sui-ssr.

Reply:
Can you put an example of use sui-deployer with sui-ssr ?!

Reply:
Can can't do auth with now without actually modifying the package.json if you use serve. In case of sui-ssr we would need to implement it in server.js. I don't want to couple sui-deploy to any file structure or code. And @carlesnunez already implement the auth thing in sui-ssr.
We are deploying a folder no matter who created it.
Here you have an example: https://github.schibsted.io/scmspain/frontend-mt--web-app/pull/295/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R36